### DISCUSSION TITLE
How to disable Light temperature triggering light to turn on

### DISCUSSION THREAD

**Discussion by humanBeer** [piitaya/lovelace-mushroom#924] [Q&A]
Anyway to not turn on devices that are currently off when I move the temperature slider of a light group?

![image](https://user-images.githubusercontent.com/47748428/210890780-82cbcf6d-cc2d-45f6-b17b-7c9626b3f14a.png)
 When i change temperature slider all lights turn on:

![image](https://user-images.githubusercontent.com/47748428/210890864-add6077a-b6ba-4b2f-a15c-095e0f80ff27.png)

**Comment by piitaya** [comment]
You can't. Home Assistant turn on light/ when changing color, brightness, etc...
